argocd/assets/dashboards/argo_cd_overview.json

```diff
@@ -0,0 +1,2352 @@
+{
+    "author_name": "Datadog",
+    "description": "## Argo CD Overview\n\n### Our Argo CD dashboard gives you broad visibility into your Argo CD Clusters\n\nThis dashboard provides a high-level overview of your Argo CD clusters so that you can monitor its deployments, performance and and the overall health of the cluster.\n\n### Useful Links\n\n- [Datadog Argo CD Integration Documentation](https://docs.datadoghq.com/integrations/argocd/)\n- [Argo CD Official Documentation](https://argo-cd.readthedocs.io/en/latest/)\n\nClone this template dashboard to make changes and add your own graph widgets. (cloned)",
```

Comment:
```suggestion
    "description": "## Argo CD Overview\n\n### The Argo CD dashboard gives you broad visibility into your Argo CD Clusters\n\nThis dashboard provides a high-level overview of your Argo CD clusters so that you can monitor the deployments, performance, and overall health of a cluster.\n\n### Useful Links\n\n- [Datadog Argo CD Integration Documentation](https://docs.datadoghq.com/integrations/argocd/)\n- [Argo CD Official Documentation](https://argo-cd.readthedocs.io/en/latest/)\n\nClone this template dashboard to make changes and add your own graph widgets. (cloned)",
```